src/QsCompiler/Tests.Compiler/TestCases/LinkingTests/SyntaxTreeTrim.qs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+// Trimmer Removes Unused Callables
+namespace Microsoft.Quantum.Testing.SyntaxTreeTrimming {
+    
+    @ EntryPoint()
+    operation Main() : Unit {
+        UsedOp();
+    }
```

Comment:
(Minor)
In other files you removed the space after `@`. Here the space is present.
Is there a any specific meaning of that space?

Reply:
🤔Yah there does seem to be some inconsistency with this. I had just copied the style I had seen in other LinkingTests test cases, but elsewhere in the project I'm seeing attributes without the space. @cgranade Which of these two is the official style for Q#, and is it worth it to fix all the Q# attributes in the compiler to have the correct style in this PR?

Reply:
Looking at the style guide, it looks like the recommended style is to not have the space after the `@`. I've updated all the references in the compiler repo that I could find to not have the space.